examples/server-parallel/server.cpp

```diff
@@ -0,0 +1,884 @@
+#include "frontend.h"
+#include "common.h"
+#include "llama.h"
+
+#include "../server/httplib.h"
+#include "../server/json.hpp"
```

Comment:
Should move these two headers from `/examples/server` into `/common`

Reply:
Done!